README.md

```diff
@@ -149,38 +149,42 @@ Serverless Framework plugin to manage APIs on [WSO2 API Manager](https://wso2.co
+> | `rootContext`                         | (CANNOT BE UPDATED LATER) <br> Your API Context, which will be exposed by WSO2 API Gateway. Must be unique per Gateway Environment.                                                                                                                                                                                         |                                                                             `/myawesomeapi` |
+> | `description`                         | Free-form text                                                                                                                                                                                                                                                                                                              |                                                                            `My Awesome API` |
+> | `visibility`                          | Supports `PUBLIC` (Visible to everyone) and `PRIVATE` (Visible to current tenant). Kept in the api for backwards compatibility. Please use fields for `subscriberVisibility` and `publisherVisibilityControl`.                                                                                                              |                                                                                    `PUBLIC` |
+> | `subscriberVisibility`                | Visibility of the API in the developer portal. Supports `PUBLIC` (Visible to everyone) and `PRIVATE` (Visible to current tenant) and `RESTRICTED` (Visible to it's tenant domain and only to the user roles that you specify. You should provide the roles in the `subscriberVisibilityRoles`.).                            |                                                                                `RESTRICTED` |
+> | `subscriberVisibilityRoles`           | The user roles that are able to access the API in Store. Only applicable in combination with `subscriberVisibility`.                                                                                                                                                                                                        |                                                                                     `admin` |
+> | `publisherVisibilityControl`          | Visibility of the API in the publisher portal. Supports `NONE` (Visible to all publishers/creators) and `RESTRICTED` (Visible to publishers/creators with the roles that you specify. You should provide the roles in the `publisherVisibilityControlRoles`.).                                                              |                                                                                `RESTRICTED` |

```

Comment:
Can you allow `PRIVATE` instead of `NONE` (to be in line with how `subscriberVisibility` value possibilities exist?